### PR TITLE
CDHSH-113 - Remake edit skills dialog and add change proficiency

### PR DESCRIPTION
**Pull request by bmalghan**
Changelog
• Remade edit skill dialog
• Added ability to change skill proficiency

### CODE REVIEW

File: services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js [hunk 1]
```diff
@@ -1,16 +1,80 @@
...
+import AddIcon from "@material-ui/icons/Add";
 import "./EditSkillsDialog.scss";
+import classNames from "classnames";
 
 const EditSkillsDialog = ({databaseSkills, skills, handleCancel, handleSubmit, open}) => {
+    //Sort the Database skills and
     const sortedSkills = Object.values(databaseSkills).sort((a, b) => a.name.localeCompare(b.name));
 
-    let currentChips = skills;
+    const [skillsUpdated, setSkills] = React.useState(JSON.parse(JSON.stringify(skills)));
```
Comment (DevinSit) [inline services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js]: I don't even want to know why this exists...

File: services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js [hunk 2]
```diff
@@ -1,16 +1,80 @@
...
+    DialogTitle,
+    IconButton,
+    TextField
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/Add";
 import "./EditSkillsDialog.scss";
+import classNames from "classnames";
 
 const EditSkillsDialog = ({databaseSkills, skills, handleCancel, handleSubmit, open}) => {
+    //Sort the Database skills and
```
Comment (DevinSit) [inline services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js]: ...

File: services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js (hunk 1 above)
Comment (bmalghan) [reply services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js]: When I set it to the skills i'm sending in, it is making changes to the current state so when i click cancel, it is "updating" (only the frontend) with the new skills changes.

File: services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js (hunk 2 above)
Comment (bmalghan) [reply services/frontend/src/components/EditSkillsDialog/EditSkillsDialog.js]: I will make this change in the remove ticket

**Pull request merged**